tests/testSelectSdssImages.py

```diff
@@ -315,16 +315,13 @@ def checkExpList(self, minRa, maxRa, runExpInfoDict):
         @param[in] maxRa: maxinum RA (degrees)
         @param[in] runExpInfoDict: a dict of run: list of ExposureInfo objects
         """
-        minRaAngle = minRa * afwGeom.degrees
         maxRaAngle = maxRa * afwGeom.degrees
```

Comment:
Please consider updating this test so that the method takes Angle as an argument. I realize it may be out of scope, and it will cascade slightly (e.g. `getCoordList` will also require the same change) but it's a nice cleanup if you are willing.